qcodes/tests/drivers/keysight_b1500/b1500_driver_tests/test_b1500.py

```diff
@@ -14,6 +14,8 @@
 from qcodes.instrument_drivers.Keysight.keysightb1500.constants import \
     SlotNr, ChNr, CALResponse
 
+# pylint: disable=redefined-outer-name
```

Comment:
If you want to you can fix this by using the name field of the fixture.

```
@pytest.fixture(name="b1500")
def _make_b1500_fixture():
    ....
```

Reply:
Thanks.

Reply:
will be done in a new PR